server.js

```diff
@@ -162,6 +162,26 @@ app.get('/api/v1/weapons/:id', async (request, response) => {
   }
 });
 
+app.post('/api/v1/characters', async (request, response) => {
+  const character = request.body;
+
+  for (let requiredParameter of ['name', 'race_id', 'class_id', 'weapon_id']) {
+    if (!character[requiredParameter]) {
+      return response
+        .status(422)
+        .send({ error: `Expected format: { name: <String>, race_id: <Number>, class_id: <Number>, weapon_id: <Number> }. You're missing a "${requiredParameter}" property.` });
+    }
+  }
```

Comment:
2. Check required parameters for received object with character. If there is no required parameter entered error is returned as response with status code of `422`